src/DssSpell.sol

```diff
@@ -19,76 +19,67 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 import "dss-interfaces/dss/IlkRegistryAbstract.sol";
-import "dss-interfaces/dss/GemJoinAbstract.sol";
-import "dss-interfaces/dss/MedianAbstract.sol";
 import "dss-interfaces/ERC/GemAbstract.sol";
 
-interface Initializable {
-    function init(bytes32 ilk) external;
-}
-
 interface VatLike {
+    function Line() external view returns (uint256);
+    function file(bytes32 what, uint256 data) external;
     function ilks(bytes32 ilk) external view returns (uint256 Art, uint256 rate, uint256 spot, uint256 line, uint256 dust);
 }
 
+interface DssVestLike {
+    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
+    function file(bytes32, uint256) external;
+    function restrict(uint256) external;
+}
 interface RwaLiquidationLike {
-    function ilks(bytes32) external view returns (string memory doc, address pip, uint48 tau, uint48 toc);
-    function init(bytes32 ilk, uint256 val, string calldata doc, uint48 tau) external;
+    function ilks(bytes32) external view returns (string memory, address, uint48, uint48);
```

Comment:
Diff here looks like you removed the parameter names - try and keep them if you can (just try and match `cast interface` results as much as possible) please :)

```suggestion
    function ilks(bytes32) external view returns (string memory doc, address pip, uint48 tau, uint48 toc);
```